src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,16 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        int result = numbers.stream()
```

Comment:
```suggestion
        return numbers.stream()
```
якщо стрім генерить те значення, що нам потрібно, можна одразу можна повертати його